packaging/README-dockerhub.md

```diff
@@ -2,8 +2,6 @@
 
```

Comment:
This file should be rename, maybe to `README-container-registry.md`.

Reply:
Done, thanks!